src/applications/vaos/new-appointment/components/VAFacilityPage/NoValidVAFacilitiesV2.jsx

```diff
@@ -1,29 +1,86 @@
 import React from 'react';
 import AlertBox from '@department-of-veterans-affairs/component-library/AlertBox';
+import State from '../../../components/State';
+import FacilityPhone from '../../../components/FacilityPhone';
+import { lowerCase } from '../../../utils/formatters';
+
+export default function NoValidVAFacilities({
+  facilities,
+  sortMethod,
+  typeOfCare,
+  address,
+}) {
+  // We're showing the two closest facilities if we have an address to sort by,
+  // otherwise we're showing the first 5 facilities in alpha order
+  const facilityLimit = address?.addressLine1 ? 2 : 5;
+  const unsupportedFacilities = facilities
+    ?.filter(
+      facility =>
+        !facility.legacyVAR.directSchedulingSupported[typeOfCare.id] &&
+        !facility.legacyVAR.requestSupported[typeOfCare.id],
+    )
+    ?.slice(0, facilityLimit);
 
-export default function NoValidVAFacilities() {
   return (
     <div aria-atomic="true" aria-live="assertive">
       <AlertBox
         status="warning"
-        headline={`Your registered facilities don’t accept online scheduling for this care right now`}
+        headline="We couldn’t find a VA facility"
+        level="2"
         content={
           <>
             <p>
-              You’ll need to call your local VA health facility to schedule this
-              appointment.{' '}
-              <a
-                href="/find-locations"
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                Find a VA location
-              </a>
-            </p>
-            <p>
-              To request another appointment online, please go back and choose a
-              different type of care.
+              None of the facilities <strong>where you receive care</strong>{' '}
+              accepts{' '}
+              <strong>
+                online appointments for {lowerCase(typeOfCare?.name)}
+              </strong>
+              .
             </p>
+            <h3 className="vads-u-font-size--h4" id="what_you_can_do">
+              What you can do
+            </h3>
+            <ul aria-labelledby="what_you_can_do">
+              {unsupportedFacilities?.length > 0 && (
```

Comment:
There should always been unsupported facilities listed here, but if there aren't, we'll still show the link to the FL